### PR TITLE
document camunda help center

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/3816.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

## Adding or removing pages requires extra steps.

- [x] I included my new page in the sidebar file(s).
- [ ] I added a redirect for a renamed or deleted page to the .htaccess file.

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Comment by christinaausley** [comment]
@HanselIdes Open to feedback on where you'd like this placed in the sidebar 👍

File: docs/guides/help-center.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+### Your recommendations
+
+As mentioned, the **Your recommendations** tab populates a list of recommended resources and next steps based on your sign-up or in-app onboarding survey.
+
+These resources may link to additional material in the Console, documentation, Camunda Academy, or blog.
+
+### Automation project guide
+
+The **Automation project guide** tab provides a set of recommended documentation pages for each phase of an automation project with Camunda SaaS. For example, a set of pages for getting started, for designing and implementing, for testing, and for going live or improving.
```
Comment (HanselIdes) [inline docs/guides/help-center.md:41]:
```suggestion
The **Automation project guide** tab provides a set of recommended documentation pages for each phase of an automation project with Camunda SaaS. For example, a set of pages for getting started, for designing and implementing, for testing, and for going live or improving your process.
```

File: docs/guides/help-center.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--cloud">Camunda 8 SaaS only</span>
+<span class="badge badge--medium">Time estimate: 10 minutes</span>
+
+Camunda 8 SaaS offers a Help Center to all users, where you can access additional documentation, step through various use cases in Camunda, share your feedback, and more.
+
+## Visiting the Help Center
+
+You can visit the Help Center directly from your Camunda 8 account. From Console or Modeler, click the **Open Help Center** icon in the top right corner of the navigation:
```
Comment (HanselIdes) [inline docs/guides/help-center.md:17]:
```suggestion
You can visit the Help Center directly from your Camunda 8 account. From Console or Modeler, click the **Open Help Center** icon in the top right corner of the navigation bar:
```

File: docs/guides/help-center.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+![question mark icon to open the help center](./img/open-help-center.png)
+
+This launches a modal for the Help Center, where you are taken to the **Your recommendations** tab.
+
+## New user survey
+
+At the top of **Your recommendations**, you may note an info box prompting you to complete the new user survey for a list of custom recommendations for additional learning.
+
+Click **Resume Survey** if you would like custom recommendations. Here, you may add additional details about your profile, goals, use case, and experience level. Fill out these details and click **Next**. Your recommendations will automatically update based on your responses.
```
Comment (HanselIdes) [inline docs/guides/help-center.md:27]:
```suggestion
Click **Resume survey** if you would like custom recommendations. Here, you may add additional details about your profile, goals, use case, and experience level. Fill out these details and click **Next**. Your recommendations will automatically update based on your responses.
```
Requesting the fix here: https://camunda.slack.com/archives/C049YH1C202/p1716549076586669
I'm comfortable with a slight inconsistency in the docs in the short term, so please don't wait to push this

File: docs/guides/help-center.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Click **Retake the survey** at any time for a new list of recommendations.
+
+## Using the Help Center
+
+### Your recommendations
+
+As mentioned, the **Your recommendations** tab populates a list of recommended resources and next steps based on your sign-up or in-app onboarding survey.
+
+These resources may link to additional material in the Console, documentation, Camunda Academy, or blog.
```
Comment (HanselIdes) [inline docs/guides/help-center.md:37]:
```suggestion
These resources may link to additional material in the Console or Camunda's Academy, documentation, or blog.
```

**Review by HanselIdes — APPROVED** [review]
LGTM - I suggested some minor changes
If you update them, you don't have to re-request a review

**Comment by HanselIdes** [comment]
> @HanselIdes Open to feedback on where you'd like this placed in the sidebar 👍

I think after Create an Account is the right place, because the user is presented the survey during the signup process
image

File: docs/guides/help-center.md (hunk 4 above)
Comment (akeller) [reply docs/guides/help-center.md:37]: Is it not "Camunda Academy"?

File: versioned_docs/version-8.5/guides/help-center.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--cloud">Camunda 8 SaaS only</span>
+<span class="badge badge--medium">Time estimate: 10 minutes</span>
+
+Camunda 8 SaaS offers a Help Center to all users, where you can access additional documentation, step through various use cases in Camunda, share your feedback, and more.
+
+## Visiting the Help Center
+
+You can visit the Help Center directly from your Camunda 8 account. From Console or Modeler, click the **Open Help Center** icon in the top right corner of the navigation:
```
Comment (akeller) [inline versioned_docs/version-8.5/guides/help-center.md:17]:
```suggestion
You can visit the Help Center directly from your Camunda 8 account. From Console or Modeler, click the **Open Help Center** icon in the top right corner of the navigation bar:
```

File: versioned_docs/version-8.5/guides/help-center.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+![question mark icon to open the help center](./img/open-help-center.png)
+
+This launches a modal for the Help Center, where you are taken to the **Your recommendations** tab.
+
+## New user survey
+
+At the top of **Your recommendations**, you may note an info box prompting you to complete the new user survey for a list of custom recommendations for additional learning.
+
+Click **Resume Survey** if you would like custom recommendations. Here, you may add additional details about your profile, goals, use case, and experience level. Fill out these details and click **Next**. Your recommendations will automatically update based on your responses.
```
Comment (akeller) [inline versioned_docs/version-8.5/guides/help-center.md:27]:
```suggestion
Click **Resume survey** if you would like custom recommendations. Here, you may add additional details about your profile, goals, use case, and experience level. Fill out these details and click **Next**. Your recommendations will automatically update based on your responses.
```

File: docs/guides/help-center.md (hunk 4 above)
Comment (akeller) [reply docs/guides/help-center.md:37]: Or was this intended to change the ordering?

File: versioned_docs/version-8.5/guides/help-center.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+### Your recommendations
+
+As mentioned, the **Your recommendations** tab populates a list of recommended resources and next steps based on your sign-up or in-app onboarding survey.
+
+These resources may link to additional material in the Console, documentation, Camunda Academy, or blog.
+
+### Automation project guide
+
+The **Automation project guide** tab provides a set of recommended documentation pages for each phase of an automation project with Camunda SaaS. For example, a set of pages for getting started, for designing and implementing, for testing, and for going live or improving.
```
Comment (akeller) [inline versioned_docs/version-8.5/guides/help-center.md:41]:
```suggestion
The **Automation project guide** tab provides a set of recommended documentation pages for each phase of an automation project with Camunda SaaS. For example, a set of pages for getting started, for designing and implementing, for testing, and for going live or improving your process.
```

**Review by akeller — APPROVED** [review]
General feedback - I'm surprised we need to document this unless we are trying to surface this in our AI tools and for general customer awareness without signing up.

File: docs/guides/help-center.md (hunk 4 above)
Comment (HanselIdes) [reply docs/guides/help-center.md:37]: > Or was this intended to change the ordering?

On second thought, it's a bad change.

**Review by HanselIdes — APPROVED** [review]

**Pull request merged**